src/wallet/wallet.cpp

```diff
@@ -4713,6 +4713,31 @@ void CWallet::GetScriptForMining(std::shared_ptr<CReserveScript> &script)
     script->reserveScript = CScript() << ToByteVector(pubkey) << OP_CHECKSIG;
 }
 
+void CWallet::GetScriptForMining(std::shared_ptr<CReserveScript> &script)
```

Comment:
See other comments, this should only load it if it exists; the validity checks should be on startup so they know right when they execute veild or veil-qt that they have a mistake in their -miningaddress, and not have to wait until later (or in the case of veild, not even notice that they didn't) get what they wanted.  I.e. find where it's checked in veild.cpp and qt/veil.cpp and add the additional checks in there.  We shouldn't fall over to do something they didn't intend when they used the flag incorrectly.

Reply:
Done.